src/models/converters/transactions/tests/is_cancellation.rs

```diff
@@ -0,0 +1,107 @@
+use super::super::chrono::Utc;
+use crate::models::backend::transactions::MultisigTransaction;
+use crate::models::commons::Operation;
+
```

Comment:
Can we add another test where `safeTxGas` is set and check that it returns `true`?